### PR TITLE
RQ v1.0!

### PR DESCRIPTION
**Pull request by selwin**

### CODE REVIEW

File: .pytest_cache/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
```
Comment (samuelcolvin) [inline .pytest_cache/README.md]: This needs to be removed.

File: rq/worker.py [hunk 2]
```diff
@@ -502,6 +481,21 @@ def work(self, burst=False, logging_level="INFO", date_format=DEFAULT_LOGGING_DA
...
+                    raise
+
+                except:  # noqa
+                    self.log.error(
+                        """
+                        Worker %s encountered an unhandled exception.\n
+                        Stacktrace:\n %s\n\n.
+                        Worker %s quitting...
+                        """,
+                        self.name, traceback.format_exc(), self.name
```
Comment (samuelcolvin) [inline rq/worker.py]: maybe `exc_info=True` instead of including the traceback in the message?

Comment (samuelcolvin) [reply rq/worker.py]: this is more canonical, and also works better on alternative log processors.

File: .pytest_cache/README.md (hunk 1 above)
Comment (selwin) [reply .pytest_cache/README.md]: Will update the PR sometime this weekend.

File: rq/worker.py (hunk 2 above)
Comment (selwin) [reply rq/worker.py]: Will try to update this.

**Comment by selwin** [comment]
@samuelcolvin updated, let me know if you spot anything else that can be improved.

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+- `job.status` has been removed. Use `job.get_status()` and `job.set_status()` instead. Thanks @selwin!
+
+- `FailedQueue` has been replaced with `FailedJobRegistry`:
+  * `get_failed_queue()` function has been removed. Please use `FailedJobRegistry(queue=queue)` instead.
+  * `move_to_failed_queue()` has been removed.
+  * RQ now provides a mechanism to automatically cleanup failed jobs. By default, failed jobs are kept for 1 year.
+  * Thanks @selwin!
+
+- RQ's custom job exception handling mechanism has also changed slightly:
+  * RQ's default exception handling mechanism (moving jobs to `FailedQueueRegistry`) can be disabled by doing `Worker(disable_default_exception_handler=True)`.
```
Comment (dralley) [inline CHANGES.md]: Should ```FailedQueueRegistry``` be ```FailedJobRegistry```?

**Comment by yanir3** [comment]
Are there any notes yet on what's new with v1.0?

**Comment by selwin** [comment]
@yanir3 yes, https://github.com/rq/rq/blob/v1.0/CHANGES.md

File: CHANGES.md (hunk 3 above)
Comment (selwin) [reply CHANGES.md]: Thanks for catching this.

**Comment by selwin** [comment]
I have merged this branch into master. Will let it sit there for a few weeks before cutting a release.

**Pull request merged**